server.js

```diff
@@ -67,7 +67,8 @@ wsServer.on('connection', function connection(ws, request) {
             switch (jsonObj.type) {
                 case 'JoinOrCreateRoom':
```

Comment:
I'm a little bit confused why there is a JoinOrCreateRoom when there are also JoinRoom and CreateRoom cases as well. What does this case add?

Reply:
This usually added for dynamically creating room or joining available rooms
It's useful for light games like pingpong where you want to just play the game with anyone available